.goreleaser.yaml

```diff
@@ -0,0 +1,57 @@
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - 
+    # Binary name
+    id: "kamaji"
+
+    # Custom environment variables to be set during the builds.
+    # Invalid environment variables will be ignored.
+    #
+    # Default: os.Environ() ++ env config section
+    # Templates: allowed (since v1.14)
+    env:
+      - CGO_ENABLED=0
+    
+    # GOOS list to build for.
+    # For more info refer to: https://golang.org/doc/install/source#environment
+    #
+    # Default: [ 'darwin', 'linux', 'windows' ]
+    goos:
+      - linux
+      - windows
+      - darwin
```

Comment:
There's no need for `darwin` support, neither `windows` I'd say.

@maxgio92 do you agree?

Reply:
Thanks for your suggestion, I will modify it later.